server/StudySharp.ApplicationServices/Queries/GetCourseByIdQuery.cs

```diff
@@ -0,0 +1,37 @@
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using StudySharp.Domain.Constants;
```

Comment:
Remove unused usings in all files
![image](https://user-images.githubusercontent.com/32524389/133981730-df45f620-79f1-4b51-88a3-ddd4a6a574fb.png)